src/ebonite/core/objects/core.py

```diff
@@ -525,3 +570,23 @@ def _generate_model_name(wrapper: ModelWrapper):
     """
     now = datetime.datetime.now()
     return '{}_model_{}'.format(wrapper.type, now.strftime('%Y%m%d_%H_%M_%S'))
+
+
+@make_string('id', 'name')
+class Image(EboniteObject):
+    def __init__(self, name: str, id: str = None,
+                 model_id: str = None, params: Dict[str, Any] = None,
+                 author: str = None, creation_date: datetime.datetime = None):
+        super().__init__(id, name, author, creation_date)
+        self.model_id = model_id
+        self.params = params or {}
+
+    @property
+    def model(self) -> Model:
+        raise AttributeError('Can\'t access model of unbound image')
```

Comment:
btw this property actually can be implemented if we have `model_id`

Reply:
Agree but we should discuss if we need this for all core objects. Doing this for just `Image` looks useless.

Reply:
Agree, can you make a task about it?

Reply:
EBNT-251